Automattic/jetpack
# AI Client: add `model` param to request helpers

On this PR, we add a `model` option to the helper that performs AI requests. For instance, `askQuestion()`,  `suggestionEventSource()`, etc.

Fixes #

## Proposed changes:

* AI Client: add `model` param to request helpers

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






For now, it's a matter of checking the dev-env compiles properly the new changes suggested here.
Stop and restart `jetpack build plugins/jetpack` should help to check that.
Also, a visual review would be nice.
